include/psp2common/kernel/iofilemgr.h

```diff
@@ -59,32 +59,46 @@ typedef enum SceIoDevType {
 	SCE_DEV_TYPE_MOUNTPT  = 0x40  //!< Mount point
 } SceIoDevType;
 
+#define SCE_DEPRECATED(x) x
```

Comment:
can you remove this? it'll remove the definition in the common/types.h

Reply:
Yeah, sorry, forgot to remove after doxygen tests. Fixed.